bindings/java/src/main/com/apple/foundationdb/FDB.java

```diff
@@ -240,6 +240,16 @@ public void enableDirectBufferQuery(boolean enabled) {
 		enableDirectBufferQueries = enabled;
 	}
 
+	/**
+	 * Determines whether getRange() queries can use {@link DirectByteBuffer} from
+	 * {@link DirectBufferPool} to copy results.
+	 *
+	 * @return {@code true} if an enabled has been selected and {@code false} otherwise
```

Comment:
```suggestion
	 * @return {@code true} if direct buffer queues have been enabled and {@code false} otherwise
```

I think. That "an" is definitely wrong

Reply:
Also "queues" to "queries" in my change suggestion.

Ironic. I could save others from typos, but not myself.